test/test_evaluation/test_train_evaluator.py

```diff
@@ -1205,6 +1294,70 @@ def test_get_splitter_cv_object(self, te_mock):
         next(cv.split(D.data['Y_train'], D.data['Y_train'],
                       groups=evaluator.resampling_strategy_args['groups']))
 
+        # KFold, regression with args
+        D.data['Y_train'] = np.array([0.0, 0.1, 0.2, 0.3, 0.4, 0.5])
+        D.info['task'] = REGRESSION
+        evaluator = TrainEvaluator()
+        evaluator.resampling_strategy = KFold
+        evaluator.resampling_strategy_args = {'folds': 4, 'shuffle': True,
+                                              'random_state': 5}
+        cv = evaluator.get_splitter(D)
+        self.assertIsInstance(cv, KFold)
+        self.assertEqual(cv.get_n_splits(
+            groups=evaluator.resampling_strategy_args['groups']), 4)
+        self.assertEqual(cv.shuffle, True)
```

Comment:
Could you please use `self.assertTrue` wherever you compare to a boolean?

Reply:
Yes, it is wiser to do it like this. thank you!